docs/product-user-guide/src/main/asciidoc/chap-red-hat-jboss-dashboard-builder.adoc

```diff
@@ -1,83 +1,173 @@
 [id='_chap_red_hat_jboss_dashboard_builder']
 = Red Hat JBoss Dashboard Builder
 
-Red Hat JBoss Dashboard Builder
- is a web-based dashboard application that provides Business Activity Monitoring (BAM) support, that is, visualization tools for monitored metrics (Key Performance Indicators, or KPIs) in real time.
-It comes integrated in the Business Central environment under the *Dashboards* menu.
+Red Hat JBoss Dashboard Builder application supports Business Activity Monitoring (BAM) using visualization tools for monitoring metrics, such as Key Performance Indicators and process statuses. With the Dashboard Builder, you can compose custom business dashboards by mixing data coming from various data sources and displaying them as graphs or tables, as well as creating custom title navigation.
 
-It comes with a dashboard that requests information from the Red Hat JBoss BPM Suite Execution Engine and provides real-time information on its runtime data; however, you can also create custom dashboards over other data sources, which leaves the application relatively standalone.
+Dashboard Builder is connected to the local JBoss BPM Suite engine by default, and acquires the data for its JBoss BPM Suite Dashboard indicators. Data sources contain custom data that can be used to create data sets. Dashboard Builder can then use these data sets to create custom perspectives. The following types of data sets are supported:
 
-[float]
-== What is Business Activity Monitoring?
+Databases::
+This uses an SQL query for obtaining data from an external database.
 
+CSV files::
+Operating over a CSV file allows the data provider to obtain all data from the file.
 
-Business Activity Monitoring (BAM) software helps to monitor business activities that take place on a computer system in real time.
-The software monitors particular metrics, such as the status of running processes, the number of invoices to be processed, processing times and other.
-It provides tools for visualization of the collected data (for example in graphs or tables).
+JAR files::
+For advanced users, JavaBeans or JAR files can be added to the Business Central distribution.
```

Comment:
"JavaBeans of JAR files" is technically incorrect. What user's have to do to use bean dataset:

1) They have to implement dashbuilder's interface org.dashbuilder.dataset.DataSetGenerator
2) They have to package this implementation into JAR file
3) They have to make this jar available on workbench's classpath (let's say by including in in business-central.war/WEB-INF/lib)

Unfortunately none of these steps is supported from UI.

Suggested reformulation:
For advanced users, who can generate DataSet's programmatically by implementing `org.dashbuilder.dataset.DataSetGenerator` and providing this implementation of (application's - business-central's) classpath.

Reply:
Changed to:

For advanced users. This requires data sets to be generated programmatically by implementing `org.dashbuilder.dataset.DataSetGenerator` and providing this implementation of the application's Business Central classpath. For more information about the Dashbuilder DataSet API, see the {URL_DEVELOPMENT_GUIDE}[{DEVELOPMENT_GUIDE}].

Reply:
One more adjustment (courtesy of another typo in my comment)

and providing this implementation of the application's Business Central classpath
-> and providing this implementation *on* the class path of Business Central